galleries/examples/ticks/scalarformatter.py

```diff
@@ -3,82 +3,30 @@
 The default tick formatter
 ==========================
 
-The example shows use of the default `.ScalarFormatter` with different
-settings.
-
-Example 1 : Default
-
-Example 2 : With no Numerical Offset
-
-Example 3 : With Mathtext
+The example shows use of the default `.ScalarFormatter` with three different
+settings: default, with mathtext, and with no numerical offset.
```

Comment:
Giving a bit more context might be helpful:

```suggestion
By default, tick labels are formatted using a `.ScalarFormatter`. This example
illustrates the effect of different settings for this formatter, which can be
configured via `~.axes.Axes.ticklabel_format`:

- default
- ``useMathText=True``: Fancy formatting of mathematical expressions
- ``useOffset=False``: Do not use offset notation. See
  `.ScalarFormatter.set_useOffset`.

```